### ISSUE TITLE
[BUG] [Linter] False positive in T001 with embedded JS curly braces

### ISSUE THREAD

**Issue by alexrudd2** [Riverside-Healthcare/djLint#606]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: MacOS Ventura
 - Python Version `3.11.0`
 - djLint Version `1.23.1`
 - template language: `jinja`

## Issue & how to reproduce
T001 False positive on
```
<button onclick="util.request(`set`, {to_set : {tags: ''}});">
```

## Contents of `.djlintrc`
```json
{
  "indent": "2",
  "max_attribute_length": "120",
  "max_line_length": "120",
  "ignore": "H021,H006,H023,H013,T028,H030,H031,H020"
}
```

Thanks for the tool, it's great!

**Comment by christopherpickering** [comment]
@alexrudd2 sorry for the delay.. should get this out in a release today.
